### DISCUSSION TITLE
possible to add the expanse / collapse row feature with the custom cell renderer ?

### DISCUSSION THREAD

**Discussion by kstang** [bosskmk/pluto_grid#524] [Q&A]
as titled, pls advise , and pls find the image below : 
![collapse](https://user-images.githubusercontent.com/8650432/180641818-ce0fb58c-5f72-49b2-92a4-4712ad70801f.png)
![expanded](https://user-images.githubusercontent.com/8650432/180641820-2385aad4-bea8-4a30-b856-51885a665266.png)

**Comment by bosskmk** [comment]
@kstang 
There is no function to expand or collapse rows.

**Reply by kstang** [reply]
possible to add in future release? I think it is very useful. I think it may implement as special column "Expandable" and we have custom renderer on top of it .

**Reply by MrFahrenheit-55** [reply]
Is this feature in the road map? Very keen to use it in my next project. I hope it releases soon!

**Reply by bosskmk** [reply]
I don't have a plan yet, but I'll try to develop it.
